utils/dc-chain/Makefile

```diff
@@ -69,12 +69,6 @@ include scripts/gcc-pass2.mk
 
 # ---- }}}}
 
-# ---- optional components {{{
-
-include scripts/gdb.mk
```

Comment:
Is that possible to first mark it "deprecated" instead of just removing it? I need to evaluate if `gdb-multiarch` embedded in MinGW-w64 can be usable, i.e., my main concern is about if this package provides a static or dynamic linked binary. If the binary is dynamic, then it won't be usable on IDEs on Windows as far as I know.

Reply:
Ideally we'd test all that before merging.

Reply:
@sizious Could you maybe have a go at mingw-w64's GDB, see if it is usable in our case with the new `kos-gdb` wrapper?